api/resource_groups.go

```diff
@@ -23,6 +23,7 @@ import (
 	"fmt"
 	"strconv"
 
+	"github.com/lacework/go-sdk/lwtime"
```

Comment:
nit: this import should be grouped at the end of the import block since it is internal